catalog/app/containers/Auth/saga.js

```diff
@@ -328,6 +345,7 @@ function* handleSignIn(
     const tokensRaw = yield call(signIn, credentials)
     const tokens = adjustTokensForLatency(tokensRaw, latency)
     const user = yield call(fetchUser, tokens)
+    yield call(setBrowseCookie, tokens)
```

Comment:
```suggestion
    yield fork(setBrowseCookie, tokens)
```

we don't need to block the execution here, `fork` sends it to the background